packages/auth/src/core/strategies/email_and_password.ts

```diff
@@ -389,5 +389,13 @@ export function signInWithEmailAndPassword(
   return signInWithCredential(
     getModularInstance(auth),
     EmailAuthProvider.credential(email, password)
-  );
+  ).catch(async error => {
+    if (
+      error.code === `auth/${AuthErrorCode.PASSWORD_DOES_NOT_MEET_REQUIREMENTS}`
+    ) {
+      await recachePasswordPolicy(auth);
+    }
+
+    return Promise.reject(error);
```

Comment:
`throw error` should be just sufficient.

Reply:
Done.